### PR TITLE
Chemistry/Hydroponics Airlock Resprites

### PR DESCRIPTION
**Pull request by Unkn0wnGh0st333**
## About the PR
Finally did the chemist and hydroponic airlocks that were missed during the massive sprite overhall

## Why / Balance
"Prioty-1" https://github.com/DeltaV-Station/Delta-v/issues/1989
yeah right nerd @MilonPL 

## Technical details
Mostly sprite work and light file changes

## Media
image

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Causes everyone to explode on an atomic level upon round start :trollface: 

**Changelog**
🆑 
tweak: Chemistry and Hydroponics doors have been readded

### CODE REVIEW

**Comment by Unkn0wnGh0st333** [comment]
Current issue, assembly version is bugged... and the chemistry one just doesn't show up:
image

**Review by deltanedas — CHANGES_REQUESTED** [review]
change the sprites in the deltav airlock groups

**Comment by Unkn0wnGh0st333** [comment]
> change the sprites in the deltav airlock groups

Uhhh is it not changed? I'm confused what you mean by this

**Comment by deltanedas** [comment]
`DeltaV/Entities/Structures/Doors/airlock_groups.yml` its for spray painter and stuff

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml [hunk 1]
```diff
@@ -76,7 +76,11 @@
   suffix: Chemistry
   components:
   - type: Sprite
-    sprite: DeltaV/Structures/Doors/Airlocks/Standard/medical.rsi # DeltaV: using medical sprite until chemistry gets resprited
+    sprite: DeltaV/Structures/Doors/Airlocks/Standard/chemistry.rsi # DeltaV - Reprite Doors
+  - type: PaintableAirlock
+    department: Medical
+  - type: Wires
+    layoutId: AirlockMedical
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml]: remove, its inherited

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml [hunk 2]
```diff
@@ -235,7 +239,12 @@
   suffix: Chemistry
   components:
   - type: Sprite
-    sprite: DeltaV/Structures/Doors/Airlocks/Glass/medical.rsi # DeltaV: using medical sprite until chemistry gets resprited
+    sprite: DeltaV/Structures/Doors/Airlocks/Glass/chemistry.rsi # DeltaV - Resprite Doors
+  - type: PaintableAirlock
+    department: Medical
+  - type: Wires
+    layoutId: AirlockMedical
+
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml]: .

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml [hunk 3]
```diff
@@ -145,7 +145,7 @@
   suffix: Hydroponics
   components:
   - type: Sprite
-    sprite: Structures/Doors/Airlocks/Standard/hydroponics.rsi
+    sprite: DeltaV/Structures/Doors/Airlocks/Standard/hydroponics.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml]: deltav comments

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml [hunk 4]
```diff
@@ -154,7 +154,7 @@
   suffix: Hydroponics, Glass
   components:
   - type: Sprite
-    sprite: Structures/Doors/Airlocks/Glass/hydroponics.rsi
+    sprite: DeltaV/Structures/Doors/Airlocks/Glass/hydroponics.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml]: .

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml [hunk 5]
```diff
@@ -195,6 +195,25 @@
...
+
+- type: entity
+  parent: AirlockAssembly
+  id: AirlockAssemblyChemistryGlass
+  suffix: Chemistry, Glass
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Doors/Airlocks/Glass/chemistry.rsi #Delta V - Resprite Doors
+    state: "assembly"
+
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml]: move these to a file in deltav folder since this doesnt exist upstream

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml (hunk 5 above)
Comment (Unkn0wnGh0st333) [reply Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml]: Wait what, chemistry and hydroponics don't exist upstream?

Edit: Ohhhh wait I get it

**Review by deltanedas — APPROVED** [review]

**Pull request merged**